Backend/router/v1/report.js

```diff
@@ -2,18 +2,20 @@ const express = require('express');
 const router = express.Router();
 const reportCtrl = require('./report.ctrl');
 
-router.use((req, res, next) => {
-  next();
-});
+module.exports = () => {
+  router.use((req, res, next) => {
+    next();
+  });
 
-router.all('/', (req, res) => {
-  res.send("this is report root");
-});
+  router.all('/', (req, res) => {
+    res.status(404).send("This is not normal route");
```

Comment:
Where do you send this "This is not normal route" message?
And how it gets displayed?